syntax = "proto3";

/* WrapperArgs consists of a specific params for a producer wrapper */
message WrapperArgs {
    string target = 1;
    string output = 2;
}

/* Config consists of params required by a producer in order to run */
message Config {
    WrapperArgs wrapperArgs = 1;
    // specific arguments for the wrapped executable
    map<string, string> toolArgs = 2;
}

